ror-demo-cluster/conf/es/readonlyrest.yml

```
@@ -1,63 +1,199 @@
-    outputs: [index]
+    outputs:    
+    - type: index
+      serializer: tech.beshu.ror.requestcontext.QueryAuditLogSerializer # tested, works OK
+
 
   access_control_rules:
 
+    ##############################
+    # local system & admin accounts
+    ##############################
     - name: "KIBANA"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)